File: php/array_front9.php
<?php
	function array_front9($arr)
	{
		$FirstFour= array_slice($arr,0,4);
		return (in_array(9,$FirstFour));
	}
	echo array_front9(array(1,2,9,3,4)) ? "pass" : "fail"; echo "\n";
	echo array_front9(array(1, 2, 3, 4, 9)) ? "fail" : "pass"; echo "\n";
	echo array_front9(array(1, 1, 2, 1, 2, 3)) ? "fail" : "pass"; echo "\n";
?>

File: ruby/array_front9.rb
def array_front9(arr)
	return arr[0..3].include?(9)
end
puts array_front9([1, 2, 9, 3, 4]) ? "pass" : "fail"
puts array_front9([1, 2, 3, 4, 9]) ? "fail" : "pass"
puts array_front9([1, 2, 3, 4, 5]) ? "fail" : "pass"